common/src/main/java/com/wynntils/models/stats/StatListOrderer.java

```diff
@@ -183,13 +188,16 @@ private static List<StatType> createDefaultOrdering(
     }
 
     private static List<StatType> createWynncraftOrdering(
+            List<SkillStatType> skillStats,
             List<MiscStatType> miscStats,
             List<DefenceStatType> defenceStats,
             List<DamageStatType> damageStats,
             List<SpellStatType> spellStats) {
         List<StatType> wynncraftOrdering = new ArrayList<>();
 
         // Wynncraft order seem to have grown a bit haphazardly
+        wynncraftOrdering.addAll(skillStats);
```

Comment:
This will match the Wynncraft ordering? I'm assuming the skill stats where sorted in some canonical skill order so this would have been okay before, so this will still be okay..?

Reply:
Wynn seems to do the ordering we have in the enum itself, with the added separator.